.github/workflows/continuous-integration.yml

```diff
@@ -113,7 +113,7 @@ jobs:
           fi
       - name: "JS Lint"
         run: |
-          if [[ -f "node_modules/.bin/eslint" && -f ".eslintrc.js" ]]; then
+          if [[ -f "node_modules/.bin/eslint" && ( -f "eslint.config.mjs" || -f ".eslintrc.js" ) ]]; then
```

Comment:
Is there any reason to have choose the `mjs` extension ? Both the migration quide https://eslint.org/docs/latest/use/configure/migration-guide and the documentation https://eslint.org/docs/latest/use/configure/configuration-files refers mainly to the `eslint.config.js` file name.

I guess most developpers will try to use the `eslint.config.js` file name at first try. If possible, we should check it in the script.

```suggestion
          if [[ -f "node_modules/.bin/eslint" && ( -f "eslint.config.js" || -f ".eslintrc.js" ) ]]; then
```

Reply:
the migration script
used here -> https://github.com/pluginsGLPI/credit/pull/145
provided by eslint pre-sets this extension

Reply:
From https://www.npmjs.com/package/@eslint/migrate-config

![image](https://github.com/user-attachments/assets/baa45f3f-bea9-48af-98b6-cf89eb85f5a4)

Reply:
we should perhaps test all extensions

https://eslint.org/docs/latest/use/configure/configuration-files

Reply:
- eslint.config.js
- eslint.config.mjs
- eslint.config.cjs

Reply:
Yes, we could test all extensions.